common/lcs/src/error.rs

```diff
@@ -13,8 +13,10 @@ pub enum Error {
     Eof,
     #[error("I/O error: {0}")]
     Io(String),
-    #[error("exceeded max sequence length")]
+    #[error("exceeded max sequence length: {0}")]
     ExceededMaxLen(usize),
+    #[error("exceeded max object depth while entering: {0}")]
+    ExceededObjectDepthLimit(&'static str),
```

Comment:
"object" depth? O_o What are those "objects" you speak of? :p

Reply:
Good question. I'm fine removing "object" everywhere.

Reply:
fixed